### PR TITLE
[PIL-462] Incative PPN screen

### PR DESCRIPTION
**Pull request by alazarevski**
Hide Settled/Unsettled Tabs when PPN is not activated/toped up

### CODE REVIEW

File: src/screens/Assets/PPNView.js [hunk 1]
```diff
@@ -347,11 +347,14 @@ class PPNView extends React.Component<Props, State> {
             color={colors.text}
             bordered
           />}
+          {!!this.props.PPNTransactions.length &&
```
Comment (jsidorenko) [inline src/screens/Assets/PPNView.js:350]:
```suggestion
          {!!PPNTransactionsMapped.length &&
```

File: src/screens/Assets/PPNView.js [hunk 2]
```diff
@@ -347,11 +347,14 @@ class PPNView extends React.Component<Props, State> {
...
           <Tabs
             tabs={historyTabs}
             wrapperStyle={{ paddingTop: 16 }}
             activeTab={activeTab}
           />
+          }
+          {!!this.props.PPNTransactions.length &&
```
Comment (jsidorenko) [inline src/screens/Assets/PPNView.js:357]:
```suggestion
          {!!PPNTransactionsMapped.length &&
```

**Comment by poocart** [comment]
Can't we just use the same logic that sits in `paymentNetwork` reducer as `isPLRTankInitialised `? It's also populated with the right value on `managePPNInitFlagAction` during import and etc.

File: src/screens/Assets/PPNView.js (hunk 1 above)
Comment (jsidorenko) [reply src/screens/Assets/PPNView.js:350]:
```suggestion
          {(!!PPNTransactionsMapped.length || availableStake > 0) &&
```

File: src/screens/Assets/PPNView.js (hunk 2 above)
Comment (jsidorenko) [reply src/screens/Assets/PPNView.js:357]:
```suggestion
          {(!!PPNTransactionsMapped.length || availableStake > 0) &&
```

**Pull request merged**